build.gradle

```diff
@@ -16,6 +16,7 @@ ext.jerseyVersion = '2.22.2'
 ext.slf4jVersion = '1.7.16'
 ext.jacksonVersion = '2.7.4'
```

Comment:
if *only* by crossdock, let's move it there, so it's less confusing at the top level.